bin/mail/2016-03-21-teacher-recruitment/generate-ops-workshop-attendees

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+require 'cdo/geocoder'
+
+# Teachers who attended Ops workshops, minus NYC
+
+teacher_query = <<eos
+  SELECT DISTINCT users.email, users.name, users.current_sign_in_ip FROM users
+    JOIN workshop_attendance ON (users.id = workshop_attendance.teacher_id)
+    JOIN segments ON (segments.id = workshop_attendance.segment_id)
+    JOIN workshops ON (workshops.id = segments.workshop_id)
+    WHERE workshops.location NOT LIKE '%NY%'
+eos
+
+results = {}
+DASHBOARD_DB.fetch(teacher_query).each do |teacher|
+  # skip teachers in NYC
+  next unless teacher[:current_sign_in_ip]
+  location = Geocoder.search(teacher[:current_sign_in_ip]).first
+  next if location && (location.city == 'New York' && location.state == 'New York')
+
+  email = teacher[:email]
+  results[email] = {email: email, name: teacher[:name]} unless results[email]
```

Comment:
I was following the [existing pattern](https://github.com/code-dot-org/code-dot-org/blob/345ce81fa9f21378e49cfc4136e23a41cc195488/bin/mail/mailing-common/mailing-list-utils.rb#L75), but there's no need here particularly because the users will be distinct. I'll remove the unless clause.